README.md

```diff
@@ -11,6 +11,29 @@ PyMC-Marketing is an open source Python package for Bayesian marketing analytics
 
 ---
 
+## Installation
+
+You may have already set up an environment (e.g. `marketing_env`) you’d like to work in. It may look something like the following:
+
+```
+mamba create -c conda-forge -n marketing_env python=3.10 matplotlib jupyterlab ipykernel seaborn pandas pymc
```

Comment:
Should probably pin the version here (5.2.0?).

Reply:
I've just tried it. I don't think 5.2.0 is available on conda-forge yet:
```
Encountered problems while solving:
  - nothing provides requested pymc 5.2.0**

The environment can't be solved, aborting the operation
```

Shall I just do `pymc>5.0`?

Reply:
done. used "pymc>=5"